modules_core/app.js

```diff
@@ -1,12 +1,18 @@
 var plugs = require('../plugs')
 var h = require('hyperscript')
+var insertCss = require('insert-css')
 
 module.exports = {
-  needs: {screen_view: 'first'},
+  needs: {
+    screen_view: 'first',
+    styles: 'first'
+  },
   gives: 'app',
   create: function (api) {
     return function () {
-      document.head.appendChild(h('style', require('../style.css.json')))
+      process.nextTick(function () {
```

Comment:
I'm a noob to this, but there might be a better way to do this if we have setImmediate around ? 
Might be worth checking how Matt did this if you haven't already